src/vue/common/OpList/OpDetails/IssuanceOpDetails.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <div class="op-details">
```

Comment:
@quinn71 about the `details` component you requested. The reason why we get rid of it eventually is that we use `email-getter` component here and we need slots in `details` and if we add them, we gonna use the same quantity of rows in comparison with the case when we do not have `details` component.

Reply:
ok